### ISSUE TITLE
Questions for the shape of the audio_embeddings

### ISSUE THREAD

**Issue by zypsjtu** [tencent-ailab/V-Express#43]
V-Express/pipelines/v_express_pipeline.py:
```python
        frame_audio_embeddings = []
        for frame_idx in range(video_length):
            start_sample = frame_idx
            end_sample = frame_idx + 2 * num_pad_audio_frames

            frame_audio_embedding = audio_embeddings[2 * start_sample:2 * (end_sample + 1), :]  # [2*num_pad+1, dim]
            frame_audio_embeddings.append(frame_audio_embedding)
        audio_embeddings = torch.stack(frame_audio_embeddings, dim=0)  # [vid_len, 2*num_pad+1, dim]
```
in my tests, the shape of the audio_embeddings is [vid_len, 2*(2*num_pad+1), dim] instead of [vid_len, 2*num_pad+1, dim] in code annotation.
